bigbluebutton-web/grails-app/conf/spring/resources.xml

```diff
@@ -85,6 +85,7 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
         <property name="captionsDir" value="${captionsDir}"/>
         <property name="recordingServiceHelper" ref="recordingServiceHelper"/>
         <property name="presentationBaseDir" value="${presentationDir}"/>
+        <property name="defaultServerUrl" value="${bigbluebutton.web.serverURL}"/>

```

Comment:
Is there an existing parameter that can be re-used here?

If not, we need to ensure this is added to the `bbb-conf --setip` command

Reply:
We are using `bigbluebutton.web.serverURL` that is defined in `bigbluebutton.propeties` and used in `paramsProcessorUtil` few lines below.